docs/cli-reference.md

```diff
@@ -299,15 +299,15 @@ Options:
   Semgrep Pro Engine options: 
     --pro                         Inter-file analysis and Pro languages
                                   (currently just Apex). Requires Semgrep Pro
-                                  Engine, contact support@r2c.dev for more
+                                  Engine, contact support@semgrep.com for more
```

Comment:
I don't think this has been updated in the actual CLI yet either - perhaps we should make sure that happens as well?

Reply:
@armchairlinguist I can make a PR to CLI...